src/components/swap/PriceImpactWarning.tsx

```diff
@@ -40,7 +39,7 @@ export default function PriceImpactWarning({ priceImpact }: PriceImpactWarningPr
               </ThemedText.DeprecatedSubHeader>
             </RowFixed>
             <ThemedText.DeprecatedLabel textAlign="right" fontSize={14} color={theme.accentFailure}>
-              {formatPriceImpact(priceImpact)}
+              {`${priceImpact.multiply(-1).toFixed(2)}%`}
```

Comment:
as a reader this is not how i would expect it to work (having to multiply by negative 1) so i do think continuing to have that named function would be nice

Reply:
To be honest, after playing around with the UI, the equivallent (self explanatory) is:
```
{`-${priceImpact.toFixed(2)}%`}
```
so I just went ahead and modified it.